auth/auth/auth.py

```diff
@@ -816,6 +816,9 @@ async def get_userinfo(request: web.Request, auth_token: str) -> UserData:
 
     userdata = await get_userinfo_from_hail_session_id(request, auth_token)
     if userdata:
+        db = request.app[AppKeys.DB]
+        await db.just_execute("UPDATE sessions SET created = NOW() WHERE session_id = %s", (auth_token,))
```

Comment:
Is this right? We store auth tokens in the database as session ids? That would be... odd. 

I suspect this is actually a naming issue within this method (it looks like it's actually being used like a session ID here, and above, and the variable should probably be named session_id instead